pango.c

```diff
@@ -93,15 +93,17 @@ PangoLayout *get_pango_layout(cairo_t *cairo, const char *font,
 
 void get_text_size(cairo_t *cairo, const char *font, int *width, int *height,
 		int *baseline, double scale, bool markup, const char *fmt, ...) {
-	char *buf = malloc(max_chars);
+	va_list args;
+
+	int size = vsnprintf(NULL, 0, fmt, args);
```

Comment:
`args` is undefined at this point, maybe `va_start` should be moved above?

Reply:
Disclaimer: I'm not a VA expert

Reply:
Yep: ```
The va_start() macro initializes ap for subsequent use by va_arg() and va_end(), and must be called first.
```

Reply:
Why is that? `vsnprintf` doesn't mutate `args`, does it?

Reply:
See `va_arg`'s manpage.

Reply:
Actually, this file doesn't even seem to be in use. Is it an artifact of moving it to a separate repo or does it need to stay?

Reply:
Oh, if it's unused, please remove it.

Reply:
Sure, I'll sweep for any others while I'm at it.